addon/components/line-clamp.js

```diff
@@ -225,7 +225,10 @@ export default class LineClampComponent extends Component {
    * @return {DidResizeModifier | null}
    */
   get didResize() {
-    return this._shouldUseNativeLineClampCSS ? null : DidResizeModifier;
+    return this._shouldUseNativeLineClampCSS ||
+      this._shouldUseNativeTextOverflowCSS
+      ? null
+      : DidResizeModifier;
```

Comment:
Minor (optional) suggestion for legibility, otherwise LGTM.
```suggestion
    const shouldUseNativeCSS =  this._shouldUseNativeLineClampCSS || this._shouldUseNativeTextOverflowCSS;
    return shouldUseNativeCSS ? null : DidResizeModifier;
```

Reply:
@drewlee Good call. Updated!